ui/src/main/java/edu/wpi/grip/ui/pipeline/StepController.java

```diff
@@ -46,6 +53,8 @@
   @FXML
   private Labeled title;
   @FXML
+  private Label elapsedTime;
```

Comment:
This label gets a little cramped with the operation icon directly above it. Not really sure what the best way would be to fix this.